contrib/go/src/python/pants/contrib/go/tasks/go_compile.py

```diff
@@ -56,6 +56,7 @@ def execute(self):
 
   def _go_install(self, target, gopath):
     args = self.get_options().build_flags.split() + [target.import_path]
+    self.context.log.info('Installing {}'.format(target.address.spec))
```

Comment:
The workunit just says `[install]`. So now it looks like this:

```
:53:29 00:05     [findbugs]
15:53:29 00:05     [go]
                   Invalidated 25 targets.
                   Installing contrib/go/examples/3rdparty/go/google.golang.org/grpc:naming
15:53:29 00:05       [install]
                     
                   Installing contrib/go/examples/src/go/cgo:cgo
15:53:30 00:06       [install]
                     
                   Installing contrib/go/examples/3rdparty/go/gopkg.in/yaml.v2:yaml.v2
15:53:31 00:07       [install]
```
 
I could change the workunit name to include the name of the target, but that's not typically what we do. I suppose because processing a single target might involve multiple workunits?